### PR TITLE
TripleLift: User sync fallback

### PR DESCRIPTION
**Pull request by davidwoodsandersen**
## Type of change
- [x] Feature

## Description of change
Adds conditional support for pixel syncing when iframe syncing is not enabled.

### CODE REVIEW

File: modules/tripleliftBidAdapter.js [hunk 1]
```diff
@@ -63,21 +63,34 @@ export const tripleliftAdapterSpec = {
...
       ibCall = utils.tryAppendQueryString(ibCall, 'cmp_cs', consentString);
     }
 
-    if (syncOptions.iframeEnabled) {
-      return [{
-        type: 'iframe',
-        url: ibCall
-      }];
-    }
+    return [{
```
Comment (willchapin) [inline modules/tripleliftBidAdapter.js]: is this return consumed by the core prebid logic? just wondering who is _actually_ doing the syncing here

File: test/spec/modules/tripleliftBidAdapter_spec.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ import { deepClone } from 'src/utils';
 import prebid from '../../../package.json';
 
 const ENDPOINT = 'https://tlx.3lift.com/header/auction?';
+const GDPR_CONSENT_STR = 'BOONm0NOONm0NABABAENAa-AAAARh7______b9_3__7_9uz_Kv_K7Vf7nnG072lPVA9LTOQ6gEaY';
```
Comment (willchapin) [inline test/spec/modules/tripleliftBidAdapter_spec.js]: thank you

File: test/spec/modules/tripleliftBidAdapter_spec.js [hunk 3]
```diff
@@ -355,12 +356,49 @@ describe('triplelift adapter', function () {
...
       expect(result).to.have.length(2);
     });
   });
+
+  describe('getUserSyncs', function() {
+    let expectedIframeSyncUrl = 'https://eb2.3lift.com/sync?gdpr=true&cmp_cs=' + GDPR_CONSENT_STR + '&';
+    let expectedImageSyncUrl = 'https://eb2.3lift.com/sync?px=1&src=prebid&gdpr=true&cmp_cs=' + GDPR_CONSENT_STR + '&';
+
+    it('returns undefined when syncing is not enabled', function() {
+      expect(tripleliftAdapterSpec.getUserSyncs({})).to.equal(undefined);
```
Comment (willchapin) [inline test/spec/modules/tripleliftBidAdapter_spec.js]: could also add the edge case where `syncOptions` is `undefined` since you guard against it in the code.

File: modules/tripleliftBidAdapter.js (hunk 1 above)
Comment (davidwoodsandersen) [reply modules/tripleliftBidAdapter.js]: Yes, `getUserSyncs` is called by the core logic [here](https://github.com/prebid/Prebid.js/blob/master/src/adapters/bidderFactory.js#L336)
